src/scipp/visualization/formatting_datagroup_html.py

```diff
@@ -86,6 +93,9 @@ def _summarize_atomic_variable(var: Any, name: str, depth: int = 0) -> str:
     if isinstance(var, Dataset | DataArray | Variable):
         parent_obj_str = "scipp"
         shape_repr = _format_shape(var)
+    if xarray_installed and isinstance(var, xr.DataArray | xr.Dataset):
+        parent_obj_str = "xarray"
+        shape_repr = _format_shape(var)
```

Comment:
Can we avoid importing Xarray entirely? What we mainly want is check for the presence of a `sizes` attribute, just like `DataGroup.__getitem__`. And the parent module could be extracted from the qualified name?

Reply:
Yes that's true, fixed now